libs/ledger/src/storage_unit/transaction_store_sync_protocol.cpp

```diff
@@ -151,18 +167,43 @@ TransactionStoreSyncProtocol::TxArray TransactionStoreSyncProtocol::PullObjects(
   return ret;
 }
 
-TransactionStoreSyncProtocol::TxArray TransactionStoreSyncProtocol::PullSpecificObjects(
-    std::vector<storage::ResourceID> const &rids)
+/**
```

Comment:
Can you summarise what major changes there are to TX sync?

Reply:
There are NO major changes to TX sync. All the changes here are interface updates essentially. I will be looking into the main tx sync issue next

Reply:
great